### PR TITLE
Add CSV

### PR DESCRIPTION
**Pull request by dinci11**

### CODE REVIEW

**Comment by dinci11** [comment]
@aklenik sorry for the late

**Comment by dinci11** [comment]
@aklenik thank you for your patience

**Comment by dinci11** [comment]
@aklenik is it in acceptable format now?

File: validator/benchmark_files/2019/NemVagyunkJokANevekben.csv [hunk 1]
```diff
@@ -0,0 +1,1082 @@
...
+START,ComputeScalar1,Pride2_Sense6,15811902158500,NemVagyunkJokANevekben,Pride2_Sense6
+END,ComputeScalar1,Pride2_Sense6,15811907115600,NemVagyunkJokANevekben,Pride2_Sense6
+QUEUE,ComputeScalar1,Pride2_Sense6,15811907139100,NemVagyunkJokANevekben,Pride2_Sense6
+START,ComputeCosine,Pride2_Sense6,15811907203900,NemVagyunkJokANevekben,Pride2_Sense6
+END,ComputeScalar3,Pride2_Sense6,15811903107500,NemVagyunkJokANevekben,Pride2_Sense6
+QUEUE,ComputeScalar3,Pride2_Sense6,15811907792500,NemVagyunkJokANevekben,Pride2_Sense6
+END,Tokenize2,Pride5_Sense5,15811905460700,NemVagyunkJokANevekben,Pride5_Sense5
+QUEUE,Tokenize2,Pride5_Sense5,15811911696300,NemVagyunkJokANevekben,Pride5_Sense5
+START,Tokenize2,Pride5_Sense6,15811911868500,NemVagyunkJokANevekben,Pride5_Sense6
+=====END=====
```
Comment (aklenik) [inline validator/benchmark_files/2019/NemVagyunkJokANevekben.csv]: Remove this, and it passes validation.

**Pull request merged**